tests/integration/tables/curl_certificate.cpp

```diff
@@ -16,39 +16,33 @@ namespace osquery {
 namespace table_tests {
 
 class curlCertificate : public testing::Test {
-  protected:
-    void SetUp() override {
-      setUpEnvironment();
-    }
+ protected:
+  void SetUp() override {
+    setUpEnvironment();
+  }
 };
 
 TEST_F(curlCertificate, test_sanity) {
   // 1. Query data
-  auto const data =
-      execute_query("select * from curl_certificate where hostname = ''");
+  auto const data = execute_query(
+      "select * from curl_certificate where hostname = \"www.github.com\"");
   // 2. Check size before validation
   // ASSERT_GE(data.size(), 0ul);
-  // ASSERT_EQ(data.size(), 1ul);
+  ASSERT_EQ(data.size(), 1ul);
   // ASSERT_EQ(data.size(), 0ul);
   // 3. Build validation map
-  // See helper.h for avaialbe flags
+  // See helper.h for available flags
   // Or use custom DataCheck object
-  // ValidatatioMap row_map = {
-  //      {"hostname", NormalType}
-  //      {"common_name", NormalType}
-  //      {"organization", NormalType}
-  //      {"organization_unit", NormalType}
-  //      {"serial_number", NormalType}
-  //      {"issuer_common_name", NormalType}
-  //      {"issuer_organization", NormalType}
-  //      {"issuer_organization_unit", NormalType}
-  //      {"valid_from", NormalType}
-  //      {"valid_to", NormalType}
-  //      {"sha256_fingerprint", NormalType}
-  //      {"sha1_fingerprint", NormalType}
-  //}
+  ValidatatioMap row_map =
+  { {"hostname", NormalType} {"common_name", NormalType} {
```

Comment:
You are missing commas between all the elements, like:
```suggestion
  { {"hostname", NormalType}, {"common_name", NormalType} {
```

Reply:
@Smjert Ahh I get it, My Bad! One thing which I find strange is that I am able to build it using Cmake on Mac with the above commas inserted on my machine but it is showing a format error here 
![image](https://user-images.githubusercontent.com/18035782/63958229-9a626680-caa7-11e9-874a-528366da60b2.png)

Reply:
Are you sure you're building tests too?
You have to pass `-DBUILD_TESTING=ON` at configure time to have them built, by default they are out of the build.

Reply:
@Smjert I did not pass that. Now I can see that it is throwing format error for linux builds but for Mac, it is giving sanity check error for our test. It seems to be that because the output of the below query:

select * from curl_certificate where hostname="www.github.com";

is zero but the assertion has been done to equalise it with 1. I feel it should have returned 1 because yes digicert is the certificate providing authority to github however this works:

select * from curl_certificate where hostname="www.facebook.com";

and provides an output